packages/graphql-yoga/__tests__/custom-serializer.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { Plugin } from '../src/plugins/types';
+import { createSchema } from '../src/schema';
+import { createYoga } from '../src/server';
+
+it('supports custom JSON serializer', async () => {
+  const useCustomSerializer: Plugin = {
+    onExecute({ setExecuteFn }) {
+      setExecuteFn(() => ({
+        stringify: () =>
```

Comment:
We could also add an assertion to check whether this function got invoked (just in case)

Reply:
👍